arch/xtensa/src/esp32s2/Kconfig

```diff
@@ -92,14 +92,16 @@ config ESP32S2_INSTRUCTION_CACHE_LINE_SIZE
 
 choice ESP32S2_DATA_CACHE_SIZE
 	prompt "Data cache size"
-	default ESP32S2_DATA_CACHE_0KB
+	default ESP32S2_DATA_CACHE_0KB if !ESP32S2_SPIRAM
+	default ESP32S2_DATA_CACHE_8KB if ESP32S2_SPIRAM
```

Comment:
Data cache initialization is still missing and it should be added to the CPU initialization routine:
https://github.com/apache/incubator-nuttx/blob/master/arch/xtensa/src/esp32s2/esp32s2_start.c#L177-L209